docs/root/start/sandboxes/grpc_bridge.rst

```diff
@@ -24,9 +24,9 @@ To build the Go gRPC service run::
   $ pwd
   envoy/examples/grpc-bridge
   $ script/bootstrap
```

Comment:
I suggest your rename `bootstrap` to `bootstrap.sh` here for consistency and updating and update the doco. Similarly rename `grpc_start` to `grpc_start.sh` and update the Dockerfile. This way all shell scripts in the directly will have .sh extensions.

Reply:
Good point, will do 👍